lib/miq_automation_engine/engine/miq_ae_service.rb

```diff
@@ -58,9 +58,12 @@ def initialize(ws, inputs = {}, body = nil, logger = $miq_ae_logger)
       self.body              = body if body
       @persist_state_hash    = ws.persist_state_hash
       @logger                = logger
+      ws.disable_rbac
```

Comment:
This feels like a race condition.

1. Can multiple `MiqAeService` objects point to the same `workspace` at the same time?
2. Is a workspace used by different `MiqAeService` at different times?
3. Are all workspace accesses channeled through `MiqAeService`?

tangent:
Do we use `MiqAeService.find` or `MiqAeServiceFront#find`.